visualization/SingVisio/webpage/resources/function.js

```diff
@@ -0,0 +1,2013 @@
+// Copyright 2021, Observable Inc.
```

Comment:
Double check this copyright information

Reply:
> Double check this copyright information

This copyright information is only for the function "Swatches()".

Reply:
Updated.